localenv/mock-account-servicing-entity/app/entry.server.tsx

```diff
@@ -27,7 +30,10 @@ async function callWithRetry(fn: () => any, depth = 0): Promise<void> {
 }
 
 if (!global.__seeded) {
-  callWithRetry(runSeed)
+  callWithRetry(async () => {
+    console.log('setting up from seed...')
+    return setupFromSeed(CONFIG, apolloClient, mockAccounts, { debug: true })
+  })
```

Comment:
same as it was before, but inlined `runSeed`. I did this because everything else from `'./lib/run_seed.server'` was moved to the new shared `mock-account-servicing-lib` and `runSeed` was the only thing left.